climakitae/explore/amy.py

```diff
@@ -203,6 +296,574 @@ def _return_diurnal(y: xr.DataArray) -> xr.DataArray:
+            index=np.arange(1, days_in_year + 1, 1),
+        )
+
+    else:
+        # Multiple warming levels AND multiple simulations
+        wl_names = [f"WL_{wl}" for wl in warming_levels]
+        sim_names = [_get_simulation_label(sim, i) for i, sim in enumerate(simulations)]
+
+        col_tuples = [
+            (hour, wl_name, sim_name)
+            for hour in hours
+            for wl_name in wl_names
+            for sim_name in sim_names
+        ]
+        multi_cols = pd.MultiIndex.from_tuples(
+            col_tuples, names=["Hour", "Warming_Level", "Simulation"]
+        )
+
+        # Stack all data horizontally
+        all_data = []
+        for hour in range(hours_per_day):
+            for wl_name in wl_names:
+                for sim_name in sim_names:
+                    key = (wl_name, sim_name)
+                    all_data.append(profile_data[key][:, hour])
+
+        all_data_array = np.column_stack(all_data)
+        df_profile = pd.DataFrame(
+            all_data_array,
+            columns=multi_cols,
+            index=np.arange(1, days_in_year + 1, 1),
+        )
+
+    # Determine which formatting function to use based on the structure
+    if not isinstance(df_profile.columns, pd.MultiIndex):
+        # Simple single-level columns
+        df_profile = _format_meteo_yr_df(df_profile)
+    else:
+        # Multi-level columns - need special formatting
+        # For now, just format the index (Day of Year)
+        year = 2024 if len(df_profile) == 366 else 2023
+        new_index = [
+            julianDay_to_date(julday, year=year, str_format="%b-%d")
+            for julday in df_profile.index
+        ]
+        df_profile.index = pd.Index(new_index, name="Day of Year")
+
+    # Preserve units information from the original data
+    if hasattr(data, "attrs") and "units" in data.attrs:
+        df_profile.attrs["units"] = data.attrs["units"]
+        df_profile.attrs["extended_description"] = data.attrs.get(
+            "extended_description", "N/A"
+        )
+        df_profile.attrs["variable_name"] = data.attrs.get("variable_id", data.name)
+
+    # Add metadata about the profile computation
+    df_profile.attrs["quantile"] = q
+    df_profile.attrs["method"] = (
+        "8760 analysis - actual data closest to quantile across 30 years"
+    )
+    df_profile.attrs["description"] = (
+        f"Climate profile computed using actual data values closest to the {q*100:.0f}th percentile of hourly data"
+    )
+
+    print(f"      ✅ Profile computation complete! Final shape: {df_profile.shape}")
+    print(
+        f"         With index: {df_profile.index.name}, columns: {df_profile.columns.names}"
+    )
+    if hasattr(data, "attrs") and "units" in data.attrs:
+        print(f"         Units: {data.attrs['units']}")
+
+    return df_profile
+
+
+def _format_meteo_yr_df(df: pd.DataFrame) -> pd.DataFrame:
+    """Format dataframe output from compute_amy and compute_severe_yr"""
```

Comment:
```suggestion
    """Format dataframe for single level WL and simulation columns"""
```

or something like that, to generalize the docstring away from deprecated functions and towards how its used in `compute_profile`